lib/foreman_inventory_upload/generators/slice.rb

```diff
@@ -36,7 +36,7 @@ def report_host(host)
         @stream.object do
           @stream.simple_field('display_name', host.name)
           @stream.simple_field('fqdn', host.fqdn)
-          @stream.simple_field('account', host.subscription_facet.pools.first.account_number.to_s)
+          @stream.simple_field('account', host.subscription_facet.pools.where.not(account_number: nil).first&.account_number&.to_s)
```

Comment:
If there is no pool with an account number, do we want a `nil` or an empty string or it doesn't matter?

Reply:
I don't think it matters. Specifically in this case, the whole `account` fields would be omitted from host.